helm/etcd-backup-operator/values.yaml

```
@@ -23,6 +23,8 @@ provider:
 aws:
   s3bucket: ""
   s3region: ""
+  enableIRSA: false
```

Should we maybe have this as an object?

```suggestion
  irsa:
    enabled: false
```

This would allow us adding more options IRSA options without always having `irsa` in the name. You could also include the role ARN there, as long as it is IRSA specific.

---

Done